src/lib.cairo

```diff
@@ -4,7 +4,4 @@ pub mod state;
 pub mod codec;
 
```

Comment:
no more merkle_tree.cairo in the lib.cairo file?

Reply:
oh yeah you made it a helper. got it

Reply:
yup